static_report/src/hooks/useAmplitudeOnMount.ts

```diff
@@ -0,0 +1,10 @@
+import { useEffect } from 'react';
+
+import { AmplitudeTrack, amplitudeTrack } from '../utils/amplitudeEvents';
```

Comment:
I just noticed this, but this naming can be a bit confusing, difference only. being that one is capitalized.

Could we rename the type as `*Event`?

Reply:
Yes, agree with you.